### ISSUE TITLE
Latest Tapiirik's super long queueing times

### ISSUE THREAD

**Issue by grzeg1** [cpfair/tapiriik#183]
Hi, lately queueing times have become longer and longer. Even few hours. So long, that I've started to manually upload activities to the services I'm using day to day. I'm a paid user and my friend using the non-paid account gets his activities synced earlier than mine at times.
Do you think you could introduce a different prioritization so premium accounts could get an instant sync when manually triggered?

**Comment by studgeek** [comment]
Yes, this would be handy. I actually pay for Tapiriik to help cover costs even though I don't use the auto sync feature (I like to edit in garmin before sync). Getting a bump in the queue would be great. I only sync every few days, but when I do I usually want it quick so I can tweak it in Strava when done.

**Comment by pterhaar** [comment]
+1

**Comment by rwa** [comment]
Agreed.  An hour plus wait even for a manual sync significantly degrades the usefulness of the service, and likely will move me onto simply using a different app which uploads to multiple databases directly.

A high priority for manual sync on a paid account is an excellent idea.

**Comment by grzeg1** [comment]
Seems this project is dying - not a single word from the developer for over 3 months :(
